src/components/ProtectedRoute.tsx

```diff
@@ -0,0 +1,11 @@
+
+import { Navigate, Outlet } from 'react-router-dom';
+import Cookies from 'js-cookie';
+
+function ProtectedRoute() {
+  const isAuthenticated = !!Cookies.get('token');
+
```

Comment:
i've seen this in App.tsx why we are using separately there?? component name is ProtectedRoute so it shoud be according to that